axelrod/tests/unit/test_version.py

```diff
@@ -0,0 +1,11 @@
+"""
+Tests the version number
+"""
+from axelrod import __version__
+import unittest
+
+expected_version = "1.11.0"
+
+class TestVersion(unittest.TestCase):
+    def test_version(self):
+        self.assertEqual(__version__, expected_version)
```

Comment:
Might as well have option 1. No harm in that and at least there's then only one place to update the version.